### DISCUSSION TITLE
website http://elastix.lumc.nl/ is down

### DISCUSSION THREAD

**Discussion by IVarha** [SuperElastix/elastix#1109] [General]
![image](https://github.com/SuperElastix/elastix/assets/8555419/6cd21b0b-a0c2-4bf6-97a0-92609fac756e)
Hello, I am trying to access website for getting a documentation, but for a few days it is down. Is it moved to some other address?

**Comment by vbpieter** [comment]
It seems to have been down for about 2 weeks, if anyone can generate and zip the doxygen files that would be great!

**Reply by IVarha** [reply]
I use [webarchive](https://web.archive.org/) to access documentation I need.
Maybe would be useful for someone.

**Reply by vbpieter** [reply]
yeah same but it's slow, would rather have the static website locally

**Reply by zed280** [reply]
I went into the same problem...

**Comment by mstaring** [comment]
A zip file with the doxygen documentation can be found via the readme: https://github.com/SuperElastix/elastix/blob/main/README.md

**Comment by N-Dekker** [comment]
🎉  Update: we have just moved the website to https://elastix.dev

With a big THANK YOU 👏 to our (LUMC/LKEB) information architect Michèle Huijberts (@mwjhhuijberts)!
